### PR TITLE
Serde

### PR DESCRIPTION
**Pull request by burrbull**
Rebased and shortened #76

### CODE REVIEW

**Comment by burrbull** [comment]
bors try

File: Cargo.toml [hunk 1]
```diff
@@ -11,8 +11,14 @@ version = "0.7.0"
 
 [features]
 unproven = []
+serde_svd = ["serde"]
```
Comment (reitermarkus) [inline Cargo.toml]: Just call this `serde`.

Comment (burrbull) [reply Cargo.toml]: I've deleted this feature as `serde` is option.

**Comment by burrbull** [comment]
Rebased once more.

**Comment by burrbull** [comment]
bors try

**Comment by therealprof** [comment]
Looks good to me per se but what is the benefit of having serde de-/serialisation here?

**Comment by burrbull** [comment]
Added https://github.com/WRansohoff/svd2json as example.

**Comment by therealprof** [comment]
Oooh, I see. That could be handy for a number of things...

**Pull request merged**